x-pack/plugins/siem/public/components/header_panel/header_panel.tsx

```diff
@@ -5,36 +5,52 @@
  */
 
 import { EuiFlexGroup, EuiFlexItem, EuiIconTip, EuiText, EuiTitle } from '@elastic/eui';
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
 import React from 'react';
 import { pure } from 'recompose';
 import styled from 'styled-components';
 
 export interface HeaderPanelProps {
+  border?: boolean;
   children?: React.ReactNode;
   subtitle?: string | React.ReactNode;
   title: string | React.ReactNode;
   tooltip?: string;
 }
 
-export const HeaderPanel = pure<HeaderPanelProps>(({ children, subtitle, title, tooltip }) => (
-  <FlexGroup alignItems="center">
-    <EuiFlexItem>
-      <EuiTitle>
-        <h2 data-test-subj="page_headline_title">
-          {title}{' '}
-          {tooltip && <EuiIconTip color="subdued" content={tooltip} position="top" size="l" />}
-        </h2>
-      </EuiTitle>
+export const HeaderPanel = pure<HeaderPanelProps>(
+  ({ border, children, subtitle, title, tooltip }) => (
+    <Header border={border}>
+      <EuiFlexGroup alignItems="center">
+        <EuiFlexItem>
+          <EuiTitle>
+            <h2 data-test-subj="page_headline_title">
+              {title}{' '}
+              {tooltip && <EuiIconTip color="subdued" content={tooltip} position="top" size="l" />}
+            </h2>
+          </EuiTitle>
 
-      <EuiText color="subdued" size="s">
-        {subtitle}
-      </EuiText>
-    </EuiFlexItem>
+          <EuiText color="subdued" size="s">
+            {subtitle}
+          </EuiText>
+        </EuiFlexItem>
 
-    {children && <EuiFlexItem grow={false}>{children}</EuiFlexItem>}
-  </FlexGroup>
-));
+        {children && <EuiFlexItem grow={false}>{children}</EuiFlexItem>}
+      </EuiFlexGroup>
+    </Header>
+  )
+);
 
-const FlexGroup = styled(EuiFlexGroup)`
-  margin-bottom: 12px;
+const Header = styled.header<{ border?: boolean }>`
+  margin-bottom: ${euiLightVars.euiSizeL};
+
+  ${props =>
+    props.border &&
+    `
+      border-bottom: ${
+        props.theme.darkMode ? euiDarkVars.euiBorderThin : euiLightVars.euiBorderThin
+      };
+      padding-bottom: ${euiLightVars.euiSizeL};
```

Comment:
Similar to above, the `euiVars` can be removed if optimized as:

``` ts
const Header = styled.header<{ border?: boolean }>`
  margin-bottom: ${({ theme }) => theme.eui.euiSizeL};

  ${props =>
    props.border &&
    `
      border-bottom: ${props.theme.eui.euiBorderThin};
      padding-bottom: ${props.theme.eui.euiSizeL};
    `}
`;
```

Reply:
👍